package.json

```diff
@@ -18,10 +18,11 @@
 	},
 	"homepage": "https://github.com/discord/discord-api-docs#readme",
 	"scripts": {
-		"build": "tsc --build",
+		"build": "tsc",
 		"lint": "eslint ci",
 		"lint:fix": "eslint ci --fix",
-		"test:links": "node dist/ci/checkLinks.js",
+		"test:links": "tsx ci/checkLinks.ts",
```

Comment:
tax is not as drop in as it claims to be.

1. It does not type check, so we need to run tsc anyways to validate types, eslint can do some of that for us, but not all. Alternatively we trust that our ides will do a good job of that (which we know wont always happen)

2. This requires changes to tsconfig to account for the fact that the code is being "bundled"

3. While we haven't set it up yet, it is possible to cache the build output, which is not possible with tsx

Reply:
re #3, the intention here is just to ensure the MDX _is valid_, not to build it.

Reply:
The point here was around my node => tsx change, which is all pretty valid 🤷 
1. good callout, added a `npm run build` as part of the lint
2. yeah, probably could change that, not hurting anything atm
3. This is for quick scripts, I am not terribly concerned with performance - ergonomics while developing is a little more important here